### PR TITLE
os/filestore: fix throttle configurations

### PR DESCRIPTION
**Pull request by dragonylffly**

### CODE REVIEW

**Comment by liewegas** [comment]
Please combine the two patches into one commit as you can't compile with only one of them applied.  Also, this will break anybody using the old config option name, right?  Probably best to keep the old one and have it override the new (more specific) values if it is >0.

**Comment by dragonylffly** [comment]
@liewegas updated

File: src/common/options.cc [hunk 1]
```diff
@@ -4533,6 +4533,22 @@ std::vector<Option> get_global_options() {
     .set_default(0)
     .set_description(""),
 
+    Option("filestore_queue_max_delay_multiple_bytes", Option::TYPE_FLOAT, Option::LEVEL_DEV)
+    .set_default(0)
+    .set_description(""),
```
Comment (tchaikov) [inline src/common/options.cc]: please document newly added options.

File: src/common/options.cc [hunk 2]
```diff
@@ -4533,6 +4533,22 @@ std::vector<Option> get_global_options() {
...
+
+    Option("filestore_queue_high_delay_multiple_bytes", Option::TYPE_FLOAT, Option::LEVEL_DEV)
```
Comment (tchaikov) [inline src/common/options.cc]: please document newly added options using `set_description()`.

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2018-05-12_08:15:46-rados-wip-kefu-testing-2018-05-12-1405-distro-basic-smithi/

**Comment by tchaikov** [comment]
retest this please.

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2018-05-14_22:14:39-rados-wip-sage3-testing-2018-05-14-1429-distro-basic-smithi/
http://pulpito.ceph.com/sage-2018-05-16_15:59:20-rados-wip-sage3-testing-2018-05-14-1429-distro-basic-smithi/

failures are http://tracker.ceph.com/issues/24150

**Pull request merged**